src/Controls/tests/TestCases.HostApp/Issues/Issue25468.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace Maui.Controls.Sample.Issues
+{
```

Comment:
Change this file name from Issue25468.cs to Issue25468.xaml.cs after uploading screenshots